src/pathfinding_service/service.py

```diff
@@ -285,56 +285,87 @@ def create_token_network_for_address(
 
     def handle_message(self, message: SignedMessage):
         if isinstance(message, UpdatePFS):
-            self.on_pfs_update(message)
+            try:
+                self.on_pfs_update(message)
+            except MessageException as x:
+                log.info(
+                    str(x),
+                    chain_id=message.canonical_identifier.chain_identifier,
+                    token_network_address=message.canonical_identifier.token_network_address,
+                    channel_identifier=message.canonical_identifier.channel_identifier,
+                    updating_capacity=message.updating_capacity,
+                    other_capacity=message.updating_capacity,
+                )
         else:
             log.info('Ignoring unknown message type')
 
     def on_pfs_update(self, message: UpdatePFS):
+        token_network_address = to_checksum_address(
```

Comment:
this should fit in one line

Reply:
to me it looks like it would be 104 chars in the line then

Reply:
then nevermind